docker/Dockerfile

```diff
@@ -114,25 +102,24 @@ RUN echo "## Update and install packages" \
         libidn11 \
         cmake \
         openjdk-11-jre-headless \
-    \
     # Standard Cleanup on Debian images
-    &&  apt-get -y clean \
-    &&  apt-get -y autoclean \
-    &&  apt-get -y autoremove \
-    &&  rm -rf /var/lib/apt/lists/* \
-    &&  rm -rf /var/cache/debconf/*-old \
-    &&  rm -rf /usr/share/doc/* \
-    &&  rm -rf /usr/share/man/?? \
-    &&  rm -rf /usr/share/man/??_* \
-&&  echo "## Done"
+    && apt-get -y clean \
+    && apt-get -y autoclean \
+    && apt-get -y autoremove \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /var/cache/debconf/*-old \
+    && rm -rf /usr/share/doc/* \
+    && rm -rf /usr/share/man/?? \
+    && rm -rf /usr/share/man/??_* \
+    && echo "## Done"
 
 # ------------------------------------------------------------------------------
 # Internal test suite of tools that this image provides
 COPY test_dockerimage.sh /emsdk/
 
 RUN echo "## Internal Testing of image" \
-    &&  /emsdk/test_dockerimage.sh  \
-&&  echo "## Done"
+    && ./emsdk/test_dockerimage.sh \
```

Comment:
Why add the leading `.` here?  Seems like we for sure want to find emsdk in the root of the filesystem no?

Reply:
Ah, added by mistake. Commit https://github.com/emscripten-core/emsdk/pull/569/commits/21d3e4756a516ce400405a0784f0d31715170469 should fix this.